.ci/integration_test.groovy

```diff
@@ -110,14 +110,14 @@ def call(ghprbActualCommit, ghprbCommentBody, ghprbPullId, ghprbPullTitle, ghprb
 
     def label = "regression-test-tispark"
 
-    podTemplate(name: label, label: label, instanceCap: 10, idleMinutes: 10, containers: [
+    podTemplate(name: label, label: label, instanceCap: 20, idleMinutes: 10, containers: [
             containerTemplate(name: 'golang', image: 'hub.pingcap.net/jenkins/centos7_golang-1.12:cached',
                     envVars: [
                             envVar(key: 'DOCKER_HOST', value: 'tcp://localhost:2375'),
                     ], alwaysPullImage: true, ttyEnabled: true, command: 'cat'),
             containerTemplate(name: 'java', image: 'hub.pingcap.net/jenkins/centos7_golang-1.13_java:cached',
-                    resourceRequestCpu: '8000m',
-                    resourceRequestMemory: '16Gi',
+                    resourceRequestCpu: '4000m',
```

Comment:
Would you mind explain why change these parameters?

Reply:
Because the resource is limited. if request for too much cpu, the pod can't be scheduled for a while